app/selectors/post_list.js

```diff
@@ -80,3 +80,26 @@ export function makePreparePostIdsForPostList() {
         }
     );
 }
+
+export function makePreparePostIdsForSearch() {
+    const getMyPosts = makeGetPostsForIds();
+
+    return createIdsSelector(
+        (state, postIds) => getMyPosts(state, postIds),
+        (posts) => {
+            const out = [];
+
+            // Iterating through the posts from oldest to newest
+            for (let i = posts.length - 1; i >= 0; i--) {
+                const post = posts[i];
+                const postDate = new Date(post.create_at);
+
+                out.push(post.id);
+                out.push(DATE_LINE + postDate.toString());
```

Comment:
If you have more than one result on the same date you’ll get more than one date separator